pycrostates/cluster/_base.py

```diff
@@ -207,6 +223,7 @@ def fit(
         info = inst.info
 
         # picks
+        self._check_picks(inst, picks)
```

Comment:
```suggestion
        _BaseCluster._check_picks(inst.info, picks)
```

Syntax for `staticmethod`.